lib/awful/widget/calendar_popup.lua

```diff
@@ -246,9 +246,12 @@ end
 --
 -- @param widget Widget to attach the calendar
 -- @tparam[opt="tr"] string position Two characters string defining the position on the screen
+-- @tparam[opt={}] table args Additional options
+-- @tparam[opt=false] bool args.on_hover Show popup while mouse hover
```

Comment:
in this calendar all parts are made like widgets, so it's possible to, for example, configure shape and spacing between days and so on

before i was using `cal` inside `naughty`